src/options.cpp

```diff
@@ -1491,6 +1491,11 @@ void options_manager::add_options_interface()
     "Vertical"
        );
 
+    add( "AIM_WIDTH", "interface", translate_marker( "Full screen Advanced Inventory Manager" ),
+        translate_marker( "Should Advanced Inventory Manager fit full screen, or leave sidebar visible." ),
```

Comment:
This text with a true/false option is not very clear on what will happen, maybe use strings fullscreen/sidebar visible or something?

Reply:
Rewrote description a bit. Hope it's more clear now.

Reply:
Rewrote description a bit. Hope it's more clear now.